libs/dal/src/repositories/subscriber/subscriber.repository.ts

```diff
@@ -30,6 +35,28 @@ export class SubscriberRepository extends BaseRepository<SubscriberEntity> {
       ],
     });
   }
+
+  async delete(query: FilterQuery<SubscriberEntity & Document>) {
+    const foundSubscriber = await this.findOne({
+      _applicationId: query.applicationId,
+      subscriberId: query.subscriberId,
+    });
+    if (!foundSubscriber)
+      throw new DalException(`Could not find subscriber with id ${foundSubscriber.subscriberId} to delete`);
```

Comment:
```suggestion
    if (!foundSubscriber) {
      throw new DalException(`Could not find subscriber with id ${foundSubscriber.subscriberId} to delete`);
    }
```

We try to avoid indentation syntax in js, it might easily create bugs, if you could just wrap this with a curly brace for the sake of clarity.